components/admin_console/database_settings.jsx

```diff
@@ -37,6 +37,7 @@ export default class DatabaseSettings extends AdminSettings {
 
     getStateFromConfig(config) {
         return {
+            connMaxLifetimeMilliseconds: config.SqlSettings.ConnMaxLifetimeMilliseconds,
```

Comment:
really minor nit: maybe order this after query timeout to match the display order?

Reply:
Good catch! Updated